ui/tests/acceptance/pki/configuration-test.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, currentURL, fillIn, visit } from '@ember/test-helpers';
+import { v4 as uuidv4 } from 'uuid';
+
+import authPage from 'vault/tests/pages/auth';
+import logout from 'vault/tests/pages/logout';
+import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
+import { runCommands } from 'vault/tests/helpers/pki/pki-run-commands';
+import { SELECTORS as S } from 'vault/tests/helpers/pki/workflow';
```

Comment:
NIT: When I came across "S" in the test it wasn't clear what it was. Maybe a word that gives some context?